quest_test/test_alias.py

```diff
@@ -0,0 +1,124 @@
+import asyncio
+import pytest
+import logging
+
+from quest import WorkflowManager
+from quest.persistence import InMemoryBlobStorage, PersistentHistory
+from src.quest import queue, Historian
+from src.quest.manager_wrappers import alias
+from utils import timeout
+
+
+@pytest.mark.asyncio
+@timeout(3)
+async def test_alias():
+    first_pause = asyncio.Event()
+    second_pause = asyncio.Event()
+    data = []
+
+    async def workflow():
+        async with queue('data', None) as q:
```

Comment:
This queue doesn't seem to get properly initialized due to a `find_historian()` failing with a `LookupError` in `InternalResource.__init__()`